tests/test_schema_editor_partitioning.py

```diff
@@ -436,3 +436,57 @@ def test_schema_editor_detach_concurrently(
     assert (
         f"{model._meta.db_table}_mypartition" in db_introspection.table_names()
     )
+
+@pytest.mark.postgres_version(lt=140000)
+@pytest.mark.django_db(transaction=True)
+@pytest.mark.parametrize(
+    "method,key,add_partition_func_name,kwargs",
+    [
+        (
+            PostgresPartitioningMethod.RANGE,
+            ["timestamp"],
+            "add_range_partition_deferred",
+            {"from_values": "2019-1-1", "to_values": "2019-2-1"},
+        ),
+        (
+            PostgresPartitioningMethod.RANGE,
+            ["id"],
+            "add_range_partition_deferred",
+            {"from_values": 1, "to_values": 10},
+        ),
+    ],
+)
+def test_schema_editor_create_and_attach_and_then_detach_concurrently(
```

Comment:
According to code above, these partitions were created by regular create not by deferred create and attach. We should have a test case to confirm that it indeed works as expected.

Reply:
You're probably right, but I don't get it :(
I've specified `add_range_partition_deferred` method in the `@pytest.mark.parametrize` decorator to use exactly create table and then attach.

Reply:
Discussed offline. Tests were improved to satisfy our criteria.